### PR TITLE
December Talk Proposal!

### PR DESCRIPTION
**Pull request by lita**
**Feel free to delete this message before submitting your PR.**

If you're submitting a PR for a talk or a performance, here's the format you should follow:

```yml
- 7:45 PM:
    type: Talk
    emoji: 📞
    title: |
      Your Title Here
    description: |
      Your description here.
    person:
      name: Foo Bar
      twitter: foobar
```

### CODE REVIEW

File: models/calendar.yml [hunk 1]
```diff
@@ -236,7 +236,20 @@
     - 8:45 PM:
         type: Talk
         emoji: 🚗
-        title: TBD
+        title: | 
+        Become a detective! Invesgiate how to use poorly documented APIs to ship faster!
```
Comment (billyroh) [inline models/calendar.yml]: @lita would you mind indenting this line? Looks like that's why the test is failing 😅

Also, we'd suggest making your title a little shorter! (<= 10 words)

File: models/calendar.yml [hunk 2]
```diff
@@ -236,7 +236,20 @@
...
+        description: |
+        You’re so pumped. You found an API that will bring
```
Comment (billyroh) [inline models/calendar.yml]: And it looks like this block of text (242-252) needs to be indented once more as well!

File: models/calendar.yml (hunk 1 above)
Comment (lita) [inline models/calendar.yml]: no worries! Will fix.

File: models/calendar.yml (hunk 2 above)
Comment (lita) [inline models/calendar.yml]: awesome! thanks for the catch.

**Pull request merged**